Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
```

Comment:
pipenv isn't really the best choice for PyPI packages - as it forces you to maintain a separate list of deps 'setup.py' and 'Pipfile'

Reply:
I suppose this is just what I am used to working under now.  I'm indifferent to either since this project doesn't have any actual dependencies outside of testing, which wouldn't appear in the `setup.py` anyway.  It just makes the easiest way to manage the pipeline for me.  I'm happy to exclude and put in the `.gitignore`, too.

Reply:
Just noticed I could have installed these as dev-packages instead